### PR TITLE
A few more export/import polishments

### PR DESCRIPTION
**Pull request by lferran**
### Description
- cleaning up unused status
- tearing down nats first (as subscribers may depend on storage, index and maindb driver utils)
- put tasks under feature flag

### How was this PR tested?
existing tests

### CODE REVIEW

File: nucliadb/nucliadb/export_import/importer.py [hunk 1]
```diff
@@ -66,6 +66,8 @@ async def import_kb(
         if item_type == ExportedItemType.RESOURCE:
             bm = cast(writer_pb2.BrokerMessage, data)
             await import_broker_message(context, kbid, bm)
+            if metadata:
```
Comment (tarekziade) [inline nucliadb/nucliadb/export_import/importer.py:69]: `if metadata is not None:`

File: nucliadb/nucliadb/export_import/importer.py [hunk 2]
```diff
@@ -85,10 +87,14 @@ async def import_kb(
             continue
 
         if metadata and count % 10 == 0:
-            # Update metadata every 10 items
+            # Save checkpoint in metadata every 10 items
             metadata.read_bytes = stream_reader.read_bytes
             await dm.set_metadata("import", metadata)
 
+    if metadata:
```
Comment (tarekziade) [inline nucliadb/nucliadb/export_import/importer.py:94]: if metadata is not None

File: nucliadb/nucliadb/tasks/consumer.py [hunk 3]
```diff
@@ -120,7 +120,7 @@ async def subscription_worker(self, msg: Msg):
                     },
                 )
                 # Nak the message to retry
-                await asyncio.sleep(1)
+                await asyncio.sleep(2)
```
Comment (tarekziade) [inline nucliadb/nucliadb/tasks/consumer.py:123]: what's the logic behind this change? in any case, could be a CONST

**Review by tarekziade — APPROVED** [review]
looks good just a few nits

File: nucliadb/nucliadb/common/context/__init__.py [hunk 4]
```diff
@@ -84,11 +84,11 @@ async def finalize(self) -> None:
         if not self._initialized:
             return
 
+        await stop_nats_manager()
```
Comment (tarekziade) [inline nucliadb/nucliadb/common/context/__init__.py:87]: nit: in theory, the tear down sequence should be in the reverse order of the creation

Comment (lferran) [reply nucliadb/nucliadb/common/context/__init__.py:87]: totally agree! otherwise we can have unexpected exceptions on service teardowns!
Changing

**Pull request merged**